docs/development/affiliated-packages.rst

```diff
@@ -501,8 +501,30 @@ replace ``CHANGES.rst`` by ``CHANGES.md`` in the instructions.
 
         git commit -m "Back to development: <next_version>"
 
-#. Check out the release commit with ``git checkout v<version>``. Run
-   ``git clean -fxd`` to remove any non-committed files, then either release with::
+#. Check out the release commit with ``git checkout v<version>``.
+   Run ``git clean -fxd`` to remove any non-committed files.
+
+#. (optional) Run the tests in a virtual environment that mocks up a
```

Comment:
The phrase "virtual environment" here could be confusing because you aren't talking specifically about "virtualenv".  Maybe just say "environment"